utils/build/docker/nodejs/express4/app.js

```diff
@@ -1,18 +1,25 @@
 "use strict";
 
-const tracer = require("dd-trace").init({
-  debug: true
+const tracer = require('dd-trace').init({
+  debug: true,
+  appsec: {
+    enabled: true
+  }
```

Comment:
This is not necessary, it should be something configurable by environment variables.

Reply:
You're right, I added it because I tested the application locally before pushing it to repo and I forgot to remove it.